README.md

```diff
@@ -69,32 +69,36 @@ Also, you can of course not use some features outlined here. That's why the whol
 
 By default `RUN_PID_DIR`, where the PID files are saved, is set to `/var/run`. It is configurable in `RUN_PID_DIR` in [`borgcron.sh`](borgcron.sh#L10). Note that for the system to work, the `RUN_PID_DIR` must **exist and be writable**. This is [usually done](https://askubuntu.com/questions/303120/how-folders-created-in-var-run-on-each-reboot) by init scripts or systemd, because `/var/run` is often mounted as a tempfs, so all data is deleted at shutdown and you have to recreate the dirs at the (next) startup. Of course, this does not matter, when running the backup as root, as it can easily recreate the directory itself, then. So either:
   * change the configuration to use a dir writable by the user, or
-  * create a init.d script or systemd service file, which creates the dir
+  * create a init.d script or systemd service file, which creates the dir in `/var/run`
 
-If the dir does not exist, the backup will not run for security reasons.
+If the given dir does not exist, the backup will not run for security reasons.
 To disable this feature, set [`RUN_PID_DIR`](borgcron.sh#L10) to an empty string (`""`). This will disable the local locking system and use borg's default locking mechanism. This is useful when you run borg v1.1.0 or higher.
 
-**Attention:** This system assumes that you access your borg repo in a "single-user" (one client, one server) environment. As the locking is managed locally, you should ensure, that **only one client** is allowed to access the borg repository. Otherwise **data loss may occur**, as this script automatically breaks the remote lock in the borg repository, if it is not locked locally.
+**Attention:** This system assumes, that you access your borg repo in a "single-user" (one client, one server) environment. As the locking is managed locally, you should ensure, that **only one client** is allowed to access the borg repository. Otherwise **data loss may occur**, as this script automatically breaks the remote lock in the borg repository, if it is not locked locally.
 
-### 4. Setup MySQL dump (optional)
+### 5. Setup MySQL dump (optional)
 
-The [`databasedump.sh`](tools/databasedump.sh) script can be used to dump the database into a dir before executing the backup. You can do this either directly before running the backup by including the file in your config file, or you can setup a new cron job using another user, who is allowed to dump the databases into a dir. The cron job should, of course, be executed before executing the actual backup.
+The [`databasedump.sh`](tools/databasedump.sh) script can be used to dump your database(s) into a dir before executing the backup. You can do this either directly before running the backup by including the content of [`databasedump.sh`](tools/databasedump.sh) in your config file, or you can setup a new cron job using another user, who is allowed to dump the databases into a dir. The cron job should, of course, be executed before executing the actual backup.
```

Comment:
No you absolutely do not have to include the content of that file. Just include (aka "source it" – using the `.` or `source` command).